test/load/allocation/grpc/README.md

```diff
@@ -74,3 +74,186 @@ You can use environment variables overwrite defaults. To run only a single run o
+  --machine-type=n2-standard-16 --region us-west1
+
+gcloud container node-pools create game-servers --cluster=scenario-test \
+  --node-taints scenario-test.io/game-servers=true:NoExecute --num-nodes=1 \
+  --machine-type n2-standard-2 --no-enable-autoupgrade \
+  --region us-west1 --tags=game-server --scopes=gke-default \
+  --enable-autoscaling --max-nodes=300 --min-nodes=175
+```
+
+## Agones Modifications
+
+For the scenario tests, we modified the Agones installation in a number of ways.
+
+First, we made sure that the Agones pods would _only_ run in the Agones node
+pool by changing the node affinity in the deployments for the controller,
+allocator service, and ping service to
+`requiredDuringSchedulingIgnoredDuringExecution`.
+
+We also increased the resources for the controller and allocator service pods,
+and made sure to specify both requests and limits to ensure that the pods were
+given the highest quality of service.
+
+These changes can be applied automatically by running
+[`configure-agones.sh`](configure-agones.sh).
+
+## Fleet Setting
+
+We used the following fleet configuration:
+
+```
+apiVersion: "agones.dev/v1"
+kind: Fleet
+metadata:
+  name: scenario-test
+spec:
+  replicas: 10
+  template:
+    metadata:
+      labels:
+        gameName: simple-game-server
+    spec:
+      ports:
+      - containerPort: 7654
+        name: default
+      health:
+        initialDelaySeconds: 30
+        periodSeconds: 60
+      template:
+        spec:
+          tolerations:
+          - effect: NoExecute
+            key: scenario-test.io/game-servers
+            operator: Equal
+            value: 'true'
+          containers:
+          - name: simple-game-server
+            image: gcr.io/agones-images/simple-game-server:0.10
+            args:
+            - -automaticShutdownDelaySec=60
+            - -readyIterations=10
+            resources:
+              limits:
+                cpu: 20m
+                memory: 24Mi
+              requests:
+                cpu: 20m
+                memory: 24Mi
+```
+
+and fleet autoscaler configuration:
+
+```
+apiVersion: "autoscaling.agones.dev/v1"
+kind: FleetAutoscaler
+metadata:
+  name: fleet-autoscaler-scenario-test
+spec:
+  fleetName: scenario-test
+  policy:
+    type: Buffer
+    buffer:
+      bufferSize: 2000
+      minReplicas: 10000
+      maxReplicas: 20000
+```
+
+## Configuring the Allocator Service
+
+The allocator service uses gRPC. In order to be able to call the service, TLS
+and mTLS have to be set up. For more information visit
+[Allocator Service](https://agones.dev/site/docs/advanced/allocator-service/).
+
+## Running the test
+
+You can use the provided runScenario.sh script by providing one parameter (a
+scenario file). Two sample scenario files are included in this directory, one
+which sends a constant rate of allocations for the duration of the test and
+another that sends a variable number of allocations.
```

Comment:
What is the format for the scenario file?

Reply:
I added a few sentences describing it.